### PR TITLE
Issue 3 - image index

### PR DESCRIPTION
**Pull request by Peter-Lavigne**
Creates an index page containing all the stored images

### CODE REVIEW

**Comment by Peter-Lavigne** [comment]
[(Peter) Issue 3: Image Index](https://trello.com/c/2drvHIv9/36-peter-issue-3-image-index)

File: test/controllers/images_controller_test.rb [hunk 1]
```diff
@@ -1,16 +1,36 @@
 require 'test_helper'
 
 class UsersControllerTest < ActionDispatch::IntegrationTest
```
Comment (butzhang) [inline test/controllers/images_controller_test.rb]: Ah, this should be `ImagesControllerTest`, you can make another commit to fix it.

File: app/controllers/images_controller.rb [hunk 2]
```diff
@@ -17,4 +15,8 @@ def create
   def show
     @url = Image.find(params[:id]).url
   end
+
+  def index
+    @urls = Image.all.map(&:url)
```
Comment (butzhang) [inline app/controllers/images_controller.rb]: In the issues requirements,  it says `Newest images appear first.`. You might need sort the images

File: test/controllers/images_controller_test.rb [hunk 3]
```diff
@@ -28,4 +28,9 @@ class UsersControllerTest < ActionDispatch::IntegrationTest
       } }
     end
   end
+
+  test 'should be an image on the landing page' do
```
Comment (butzhang) [inline test/controllers/images_controller_test.rb]: If you want to test there is only one image on the landing page, you should do `assert_select 'img', count: 1`
